src/API/Site/Controllers/Ads/CampaignController.php

```diff
@@ -152,6 +154,14 @@ protected function create_campaign_callback(): callable {
 					);
 				}
 
+				if ( $this->is_wc_mobile_app( $request ) ) {
+					if ( $this->is_wc_ios( $request ) ) {
+						$fields['label'] = 'wc-ios';
+					} else {
+						$fields['label'] = 'wc-android';
+					}
+				}
```

Comment:
PHP will do some optimization but if we follow the code path this will technically fetch the user agent header and then check if it contains the string `wc-ios` twice. If we aren't going to optimize it so it only needs to compare the header once, then wouldn't it look slightly cleaner to check it like this:

```suggestion
				if ( $this->is_wc_ios( $request ) ) {
					$fields['label'] = 'wc-ios';
				} elseif ( $this->is_wc_android( $request ) ) {
					$fields['label'] = 'wc-android';
				}
```